extensions/amp-accordion/1.0/accordion.js

```diff
@@ -97,16 +103,87 @@ export function Accordion({
     [expandSingleSection]
   );
 
+  const isExpanded = useCallback(
+    (id, defaultExpanded) => expandedMap[id] ?? defaultExpanded,
+    [expandedMap]
+  );
+
+  const toggle = useCallback(
```

Comment:
Suggestion: `toggle`, `collapse`, and `expand` all look nearly identical in their implementations, is it possible to dedup some of this logic?

Reply:
Thanks for the suggestion.  I also felt this way initially, and considered writing some type of generic function in which you can pass in the specific behavior per each different action.  Ultimately, I decided this looks a lot cleaner in terms of readability.  Especially since we actually do have 6 distinct behaviors (2 for each of the 3 actions).

Also as a bonus, later if we have to re-factor the behavior of `expandSingleSection` I believe it's easier in the current format.

Thoughts?

Reply:
Since I haven't heard back, I'm going to go ahead and merge this PR.  But if you have additional thoughts we can address in a follow up.  Thanks!

Reply:
SGTM

Reply:
Great!